internal/svcctl/svcctl.go

```diff
@@ -21,11 +21,12 @@ import (
 	"github.com/ActiveState/cli/internal/locale"
 	"github.com/ActiveState/cli/internal/logging"
 	"github.com/ActiveState/cli/internal/osutils"
+	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/profile"
 )
 
 var (
-	pingRetryIterations = 32
+	pingRetryIterations = 45
```

Comment:
Using VERBOSE=true and trial and error I found this iteration gives a 31s timeout, which is close to the 30s target in the ACs.